pkg/common/message/message.go

```diff
@@ -27,7 +27,7 @@ const (
 
 	ScaleTargetNotFoundMsg = "Target resource doesn't exist"
 
-	ScaleTargetNoSubresourceMsg = "Target resource doesn't expose /scale subresource"
+	ScaleTargetNoSubresourceMsg = "Target resource doesn't expose / scale subresource"
```

Comment:
I'm not sure about this change, the resource is `/scale`

Reply:
My mistake! I corrected it right away.